module/VuFindApi/src/VuFindApi/Controller/ApiController.php

```diff
@@ -124,7 +133,6 @@ protected function getApiSpecs()
             }
         }
 
-        // Return the fragment without the enclosing curly brackets
-        return substr(trim(json_encode($results, JSON_PRETTY_PRINT)), 1, -1);
+        return $results;
```

Comment:
I'm having a little trouble following why we were able to remove the brace-stripping logic in the refactoring, but I assume it works, so I'm probably just missing something.

Reply:
That was because previously the returned string was added in the main JSON response as a string, so it couldn't have had the surrounding braces. Now that it's handled as an array, that hackish approach is no longer necessary.

Reply:
Makes sense, thanks for clarifying!